packages/@lwc/engine-server/src/__tests__/fixtures.spec.ts

```diff
@@ -161,17 +161,24 @@ function testFixtures() {
                 },
             });
 
-            const result = lwcEngineServer!.renderComponent(
-                module!.tagName,
-                module!.default,
-                config.props || {}
-            );
+            let result;
+            let err;
+            try {
+                result = lwcEngineServer!.renderComponent(
+                    module!.tagName,
+                    module!.default,
+                    config.props || {}
+                );
+            } catch (_err: any) {
+                err = _err.message;
+            }
             features.forEach((flag) => {
                 lwcEngineServer!.setFeatureFlagForTest(flag, false);
             });
 
             return {
-                'expected.html': formatHTML(result),
+                'expected.html': result ? formatHTML(result) : undefined,
+                'expected-error.txt': err,
```

Comment:
In other fixture tests we use `error.json` (with the error message and `loc` line/column offsets), but it's not a big deal.

Reply:
I was thinking about this too. But because this is a runtime error and not a compilation error, attributing to a LOC doesn't really work.

Originally, I included `err.stack` (so that the problematic call of `contextProvider(notAnElm, { ... })` would be captured & reported. But the stack includes file paths specific to the computer on which the test is running. It also isn't very meaningful, as the stack trace might change in the future and that probably shouldn't break the test.

For consistency with the `error.json` elsewhere, perhaps I could rename to `error.txt`?

Reply:
`error.txt` is good with me.